api/mgo/delete.go

```diff
@@ -11,17 +11,17 @@ import (
 
 // Delete contains the methods for the delete operation
 type Delete struct {
-	ctx    context.Context
-	meta   *proto.Meta
-	op     string
-	find   utils.M
-	config *config.Config
+	ctx         context.Context
+	meta        *proto.Meta
+	op, db, col string
+	find        utils.M
+	config      *config.Config
 }
 
 func initDelete(ctx context.Context, db, col, op string, config *config.Config) *Delete {
 	m := &proto.Meta{Col: col, DbType: db, Project: config.Project, Token: config.Token}
 	f := make(utils.M)
-	return &Delete{ctx, m, op, f, config}
+	return &Delete{ctx, m, op, db, col, f, config}
```

Comment:
Why store db and collection again? Its already there in the meta

Reply:
Oh. Right